olympics.md

```diff
@@ -0,0 +1,25 @@
+--
+title: All the medal winners in the history of the Summer Olympic Games and more 
+description: A compilation of all the athletes who won at least one medal in every SOG, from Athens 1896 to Paris 2024, with emphasis on the most recent Games
+keywords: olympics, olympic games, paris 2024, medal winners
+date: 01/10/2024
```

Comment:
this looks like non-standard format for dates, could you do `YYYY-MM-DD` please?